src/aquarium-optimized/dawn/ContextDawn.cpp

```diff
@@ -898,7 +900,6 @@ void ContextDawn::Flush() {
 }
 
 void ContextDawn::Terminate() {
-  glfwTerminate();
```

Comment:
Do we need an empty function here?

Reply:
The only purpose is to have something symmetric with `initialize()`. But in the long term I doubt a separate `initialize()` method is needed besides the constructor.